### PR TITLE
[ADD] Add spanish translation

### PR DESCRIPTION
**Pull request by eantones**
There's no Transifex project for this module.

### CODE REVIEW

File: website_sale_cart_preview/i18n/en.po [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline website_sale_cart_preview/i18n/en.po]: Remove this file

File: website_sale_cart_preview/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.5.4\n"
+"Language: Spanish\n"
+
+#. module: website_sale_cart_preview
+#. openerp-web
+#: code:addons/website_sale_cart_preview/static/src/js/website_sale_cart_preview.js:11
+#, python-format
+msgid "My Cart"
+msgstr "Mi carro"
```
Comment (pedrobaeza) [inline website_sale_cart_preview/i18n/es.po]: "Mi cesta" (el otro es de Manolo Escobar :wink:)

File: website_sale_cart_preview/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+#. module: website_sale_cart_preview
+#: view:website:website_sale_cart_preview.cart_popover
+msgid "Qty:"
+msgstr "Cant:"
```
Comment (pedrobaeza) [inline website_sale_cart_preview/i18n/es.po]: "Cant.:"

File: website_sale_cart_preview/i18n/es.po [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+#. module: website_sale_cart_preview
+#: view:website:website_sale_cart_preview.total
+msgid "Total:"
+msgstr "Total:"
+
+#. module: website_sale_cart_preview
+#: view:website:website_sale_cart_preview.cart_popover
+msgid "View Cart ("
+msgstr "Ver carro ("
```
Comment (pedrobaeza) [inline website_sale_cart_preview/i18n/es.po]: cesta

File: website_sale_cart_preview/i18n/es.po [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+#. module: website_sale_cart_preview
+#: view:website:website_sale_cart_preview.cart_popover
+msgid "Your cart is empty!"
+msgstr "¡Su carro está vacío!"
```
Comment (pedrobaeza) [inline website_sale_cart_preview/i18n/es.po]: cesta

File: website_sale_cart_preview/i18n/es.po [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+#. module: website_sale_cart_preview
+#: view:website:website_sale_cart_preview.cart_popover
+msgid "items)"
+msgstr "items)"
```
Comment (pedrobaeza) [inline website_sale_cart_preview/i18n/es.po]: elementos

**Comment by eantones** [comment]
Hola, está ahora todo correcto, es posible el merge? Gracias

**Comment by pedrobaeza** [comment]
Remove the en.po file.

Elimina el archivo en.po.

**Comment by pedrobaeza** [comment]
Please squash all commits in one, and have you signed the CLA?

**Comment by eantones** [comment]
**Please squash all commits in one**
Done

**have you signed the CLA?**
I sent an email to odoo community but I haven't received a response yet

**Comment by pedrobaeza** [comment]
:+1: 

Thanks

**Comment by pedrobaeza** [comment]
I merge directly as it's a translation

**Pull request merged**